apps/docs/i18n/es/docusaurus-plugin-content-docs/version-V3/guides/identities.mdx

```diff
@@ -37,19 +37,19 @@ values={[
 <TabItem value="npm">
 
 ```bash
-npm install @semaphore-protocol/identity
+npm install @semaphore-protocol/identity@3
 ```
 </TabItem>
 <TabItem value="yarn">
 ```bash
-yarn add @semaphore-protocol/identity
+yarn add @semaphore-protocol/identity@3
```

Comment:
This worked for you? It installed the version `3`. It worked with `yarn add @semaphore-protocol/identity@3.15.2`.

Reply:
Yes, it should install the latest version. Can you check the `node_modules/@semaphore-protocol/identity/package.json` version?

Reply:
Yes, it is the right one in node_modules but in the project package json file is `"@semaphore-protocol/identity": "3"`. If node_modules are fine, it should work fine.

Reply:
I've gone into deep a bit, `@^3.x` might be better.